stacksandqueues/app/src/test/java/stacksandqueues/AppTest.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * This Java source file was generated by the Gradle 'init' task.
+ */
+package stacksandqueues;
+
+import org.junit.Test;
+import org.junit.function.ThrowingRunnable;
+
+import java.util.EmptyStackException;
+
+import static org.junit.Assert.*;
+
+public class AppTest {
```

Comment:
I'd suggest breaking down your tests a little bit. Its great that you separated the test from a stack and a queue, but you may want to write an individual test for each method.